mpitests/test_apply.py

```diff
@@ -447,7 +445,7 @@ class TestExpval:
             qml.PauliY,
             qml.PauliZ,
             qml.Hadamard,
-            qml.Identity,
+            # qml.Identity,
```

Comment:
The 'Identity' gate always fails with MPI. @multiphaseCFD  suggest to discuss this offline with the team. :rocket:

Reply:
We have to bypass this op in the serialization step. The `Identity` gate does nothing and can be skipped from the list of ops.

Reply:
I also would recommend to add #TODOs to the PR discription

Reply:
@multiphaseCFD  Done!